### PR TITLE
Closes #665, Closes #667: Update Drupal core to 9.1.7 for SA-CORE-2021-002 (critical security update).

### LINKED ISSUE #665: Update Drupal core to 9.1.6
We need to update Quickstart to use the latest Drupal core bugfix release:
https://www.drupal.org/project/drupal/releases/9.1.6

### LINKED ISSUE #667: Drupal core security security update for SA-CORE-2021-002
## Problem/Motivation
A security advisory regarding a new release that addresses a critical security vulnerability in Drupal core has been published.  We should update our version of Drupal core ASAP.

### Describe the bug
https://www.drupal.org/sa-core-2021-002

## Proposed resolution
Update to Drupal 9.1.7

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
https://www.drupal.org/sa-core-2021-002

This also includes 9.1.6 since we haven't merged #666 yet.

## Related Issue
#665, #667

## How Has This Been Tested?

ProboCI FTW!

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by camikazegreen — APPROVED** [review]
I tested in Probo by adding a Person and a Page that have dashes in the input fields. Everything displays as expected.

**Comment by joeparsons** [comment]
I can't approve/merge this myself but I think it should be ready to merge once we have enough approvals.

**Review by trackleft — APPROVED** [review]

**Pull request merged**
